docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -179,7 +203,7 @@ The response below shows that the bootstrap index is waiting in the `hot` phase'
 It remains in this state and {ilm-init} continues to call `attempt-rollover`
 until the rollover conditions are met.
 
-[[36818c6d9f434d387819c30bd9addb14]]
+// [[36818c6d9f434d387819c30bd9addb14]]
```

Comment:
Curious: what's the purpose of this anchor?

Reply:
Those are for the client examples.